src/server/controllers/match.controller.js

```diff
@@ -99,22 +100,28 @@ export function getMentorList(req, res, next) {
     })
     .then((careerFilteredList) => {
       return new Promise((resolve, reject) => {
-        let filteredList = {
-          full: [],
-          idList: [],
-        };
+        if (req.body.expertise === undefined) {
+          resolve(careerFilteredList.full);
+        } else if (req.body.expertise.length === 0) {
+          resolve(careerFilteredList.full);
```

Comment:
If `req.body.expertise` is `undefined`, `req.body.expertise.length` causes an error.